pkg/testlib/testlib.go

```diff
@@ -463,8 +463,8 @@ func SetHardwareVersion(vmClient *vim25.Client, node *v1.Node, hardwareVersion s
 	return err
 }
 
-func CustomizeHostVersion(hostSystemId string, version string, apiVersion string) error {
-	hsRef := simulator.Map.Get(types.ManagedObjectReference{
+func CustomizeHostVersion(model *simulator.Model, hostSystemId string, version string, apiVersion string) error {
```

Comment:
I will rework this to use just context instead of model, I am also thinking it could be better to return default simulation context from the SetupSimulation function (get it out of model in that function instead of doing it afterwards where the model may not be available)